delta/plugins/blazegraph/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/model/BlazegraphViewRejection.scala

```diff
@@ -40,8 +43,10 @@ object BlazegraphViewRejection {
     * Rejection returned when attempting to update a view that doesn't exist.
     *
     * @param id the view id
+    * @param project the project it belongs to
     */
-  final case class ViewNotFound(id: Iri) extends BlazegraphViewRejection(s"View '$id' not found.")
+  final case class ViewNotFound(id: Iri, project: ProjectRef)
+      extends BlazegraphViewRejection(s"View '$id' not found in project '$project]'.")
```

Comment:
Blazegraph view would disembiguate the message with Elastic/Composite views
There is also an extra ] to remove